src/main/kotlin/racingcar/MovingJudgeRule.kt

```diff
@@ -0,0 +1,34 @@
+package racingcar
+
+enum class MovingJudgeRule(
+    private val expression: (Int) -> (Boolean),
+    private val strategy: MovingStrategy
+) {
+
+    ADVANCE_RULE(
+        expression = { number -> number >= FORWARD_STRATEGY_BOUND },
```

Comment:
람다식의 변수가 한 개 뿐이라면 별도의 ** variableName -> xxxx** 표현 없이 `it`으로 바로 람다 변수를 사용할 수 있어요! 😉 
```suggestion
        expression = { it >= FORWARD_STRATEGY_BOUND },
```

Reply:
이렇게 줄일 수 있었네요. number -> number 가 조금은 불필요해 보였는데 리뷰어님 덕분에 하나 더 배워가네요 👍